Fixing some typos and errors

NeonSpork opened the pull request:
Updates to test.py and calibrate.py
Changed hx711.py back to "pass" instead of printing "No input from hx711" to reduce the noise in the command console while running programs.

NeonSpork committed:
Update test.py

Fixing typo

NeonSpork committed:
Update calibration.py

Forgot quotes

robster0326 commented:
This what i got :(

https://ibb.co/kk2mQ0


NeonSpork <notifications@github.com> ezt írta (időpont: 2018. nov. 3., Szo,
10:11):

> Forgot quotes
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/learn2develop/hx711py/pull/5
> Commit Summary
>
>    - Update calibration.py
>
> File Changes
>
>    - *M* calibration.py
>    <https://github.com/learn2develop/hx711py/pull/5/files#diff-0> (6)
>
> Patch Links:
>
>    - https://github.com/learn2develop/hx711py/pull/5.patch
>    - https://github.com/learn2develop/hx711py/pull/5.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/learn2develop/hx711py/pull/5>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEDj7j80BmYyiKTzmBeDcLu_8H_jopl6ks5urV3JgaJpZM4YMtm_>
> .
>

NeonSpork committed:
Update calibration.py

NeonSpork committed:
Update calibration.py

NeonSpork committed:
Update hx711.py

Turning off the print about no input while it's not ready, changed back to "pass" like original. The output was too noisy.

NeonSpork commented:
Alright a few things:
* Updated hx711.py back to "pass" when the chip isn't ready. Printing "No input from hx711" was too noisy in the command console for my tastes. Agreed?
* Fixed the input prompts to correctly take the string input.
* Changed the catch-all for invalid inputs to hopefully work more smoothly than just an else clause.

@robster0326 Use the updated files from my patch branch for now, none of this is merged.
@learn2develop Can we pause the merge request while we try to iron out these bugs? :)

NeonSpork committed:
Merge pull request #1 from NeonSpork/patch-2

Update test.py

NeonSpork committed:
Merge pull request #2 from NeonSpork/patch-3

Patch 3

NeonSpork committed:
added a comment in the header that calibrartion needs to run in python 3

NeonSpork committed:
Made separate example files, one for python 3, one for python 2.7

NeonSpork committed:
Added comment in header that test.py will only run on python 3

NeonSpork committed:
Merge pull request #3 from NeonSpork/patch-3

Patch 3

NeonSpork committed:
Forgot to set the scale!

NeonSpork committed:
Merge pull request #4 from NeonSpork/master

Forgot to set_scale

NeonSpork committed:
Update calibration.py

Setting gain at 128 as default

NeonSpork committed:
Merge pull request #5 from NeonSpork/master

Update calibration.py

NeonSpork committed:
Setting gain at 128 as default

robster0326 commented:
Good morning!

It still does not work for me, after downloading the new version:


root@raspberrypi:/var/www/html/n# python3 test.py
Initializing HX711
Place something with a known weight on the scale.
Press enter when ready.
8366205
Enter weight on scale in grams: 434
Weight: -2.817254209046993g, verify that the measured weight is correct.
Provided the above result is correct the following can be used in your main
program:
Offset: 8420416.375
Scale: 19276.970046082948
Test complete. Exiting.

I have checked everything, but can't find the root cause :(

What else could i try?

Jiri Dohnalek <notifications@github.com> ezt írta (időpont: 2018. nov. 4.,
V, 11:31):

> Merged #5 <https://github.com/learn2develop/hx711py/pull/5> into master.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/learn2develop/hx711py/pull/5#event-1944758164>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AEDj7r_U58N7wlz1tGUn8qzGPHstC3tAks5ursHvgaJpZM4YMtm_>
> .
>

NeonSpork commented:
Just checking real quick, you're probably doing it right but it doesn't
hurt to ask:

You're making sure to have absolutely nothing at all on the scale until
AFTER the program says "Place something with a known weight on the scale."
right?

On Wed, Nov 7, 2018, 09:20 robster <notifications@github.com> wrote:

> Good morning!
>
> It still does not work for me, after downloading the new version:
>
>
> root@raspberrypi:/var/www/html/n# python3 test.py
> Initializing HX711
> Place something with a known weight on the scale.
> Press enter when ready.
> 8366205
> Enter weight on scale in grams: 434
> Weight: -2.817254209046993g, verify that the measured weight is correct.
> Provided the above result is correct the following can be used in your main
> program:
> Offset: 8420416.375
> Scale: 19276.970046082948
> Test complete. Exiting.
>
> I have checked everything, but can't find the root cause :(
>
> What else could i try?
>
> Jiri Dohnalek <notifications@github.com> ezt írta (időpont: 2018. nov. 4.,
> V, 11:31):
>
> > Merged #5 <https://github.com/learn2develop/hx711py/pull/5> into master.
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/learn2develop/hx711py/pull/5#event-1944758164>, or
> mute
> > the thread
> > <
> https://github.com/notifications/unsubscribe-auth/AEDj7r_U58N7wlz1tGUn8qzGPHstC3tAks5ursHvgaJpZM4YMtm_
> >
> > .
> >
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/learn2develop/hx711py/pull/5#issuecomment-436541441>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AfteJVC0tH6jUrGLsPzGrrKmcKzjAawgks5uspe6gaJpZM4YMtm_>
> .
>

robster0326 commented:
Yeps, i do everything as instructed by the program.

NeonSpork <notifications@github.com> ezt írta (időpont: 2018. nov. 7., Sze,
9:31):

> Just checking real quick, you're probably doing it right but it doesn't
> hurt to ask:
>
> You're making sure to have absolutely nothing at all on the scale until
> AFTER the program says "Place something with a known weight on the scale."
> right?
>
> On Wed, Nov 7, 2018, 09:20 robster <notifications@github.com> wrote:
>
> > Good morning!
> >
> > It still does not work for me, after downloading the new version:
> >
> >
> > root@raspberrypi:/var/www/html/n# python3 test.py
> > Initializing HX711
> > Place something with a known weight on the scale.
> > Press enter when ready.
> > 8366205
> > Enter weight on scale in grams: 434
> > Weight: -2.817254209046993g, verify that the measured weight is correct.
> > Provided the above result is correct the following can be used in your
> main
> > program:
> > Offset: 8420416.375
> > Scale: 19276.970046082948
> > Test complete. Exiting.
> >
> > I have checked everything, but can't find the root cause :(
> >
> > What else could i try?
> >
> > Jiri Dohnalek <notifications@github.com> ezt írta (időpont: 2018. nov.
> 4.,
> > V, 11:31):
> >
> > > Merged #5 <https://github.com/learn2develop/hx711py/pull/5> into
> master.
> > >
> > > —
> > > You are receiving this because you were mentioned.
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/learn2develop/hx711py/pull/5#event-1944758164>, or
> > mute
> > > the thread
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AEDj7r_U58N7wlz1tGUn8qzGPHstC3tAks5ursHvgaJpZM4YMtm_
> > >
> > > .
> > >
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/learn2develop/hx711py/pull/5#issuecomment-436541441
> >,
> > or mute the thread
> > <
> https://github.com/notifications/unsubscribe-auth/AfteJVC0tH6jUrGLsPzGrrKmcKzjAawgks5uspe6gaJpZM4YMtm_
> >
> > .
> >
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/learn2develop/hx711py/pull/5#issuecomment-436544561>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEDj7rYqndaJpLRqDtVqhxX8U-ysuo4_ks5usppKgaJpZM4YMtm_>
> .
>

robster0326 commented:
I'm using 4 strain gauges connected to a Load Combinator, which is
connected to a Load Cell Amplifier what is connected to my raspi. :( maybe
the weight is too small?

Robert Vajda <robster0326@gmail.com> ezt írta (időpont: 2018. nov. 7., Sze,
10:41):

> Yeps, i do everything as instructed by the program.
>
> NeonSpork <notifications@github.com> ezt írta (időpont: 2018. nov. 7.,
> Sze, 9:31):
>
>> Just checking real quick, you're probably doing it right but it doesn't
>> hurt to ask:
>>
>> You're making sure to have absolutely nothing at all on the scale until
>> AFTER the program says "Place something with a known weight on the scale."
>> right?
>>
>> On Wed, Nov 7, 2018, 09:20 robster <notifications@github.com> wrote:
>>
>> > Good morning!
>> >
>> > It still does not work for me, after downloading the new version:
>> >
>> >
>> > root@raspberrypi:/var/www/html/n# python3 test.py
>> > Initializing HX711
>> > Place something with a known weight on the scale.
>> > Press enter when ready.
>> > 8366205
>> > Enter weight on scale in grams: 434
>> > Weight: -2.817254209046993g, verify that the measured weight is correct.
>> > Provided the above result is correct the following can be used in your
>> main
>> > program:
>> > Offset: 8420416.375
>> > Scale: 19276.970046082948
>> > Test complete. Exiting.
>> >
>> > I have checked everything, but can't find the root cause :(
>> >
>> > What else could i try?
>> >
>> > Jiri Dohnalek <notifications@github.com> ezt írta (időpont: 2018. nov.
>> 4.,
>> > V, 11:31):
>> >
>> > > Merged #5 <https://github.com/learn2develop/hx711py/pull/5> into
>> master.
>> > >
>> > > —
>> > > You are receiving this because you were mentioned.
>> > > Reply to this email directly, view it on GitHub
>> > > <https://github.com/learn2develop/hx711py/pull/5#event-1944758164>,
>> or
>> > mute
>> > > the thread
>> > > <
>> >
>> https://github.com/notifications/unsubscribe-auth/AEDj7r_U58N7wlz1tGUn8qzGPHstC3tAks5ursHvgaJpZM4YMtm_
>> > >
>> > > .
>> > >
>> >
>> > —
>> > You are receiving this because you authored the thread.
>> > Reply to this email directly, view it on GitHub
>> > <https://github.com/learn2develop/hx711py/pull/5#issuecomment-436541441
>> >,
>> > or mute the thread
>> > <
>> https://github.com/notifications/unsubscribe-auth/AfteJVC0tH6jUrGLsPzGrrKmcKzjAawgks5uspe6gaJpZM4YMtm_
>> >
>> > .
>> >
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/learn2develop/hx711py/pull/5#issuecomment-436544561>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AEDj7rYqndaJpLRqDtVqhxX8U-ysuo4_ks5usppKgaJpZM4YMtm_>
>> .
>>
>

robster0326 commented:
If i put more weight on it, the value change, but it's still negative:

root@raspberrypi:/var/www/html/n# python3 test.py
Initializing HX711
Place something with a known weight on the scale.
Press enter when ready.
8366255
Enter weight on scale in grams: 4126
Weight: -26.822929345328347g, verify that the measured weight is correct.
Provided the above result is correct the following can be used in your main
program:
Offset: 8420312.625
Scale: 2027.6914687348522
Test complete. Exiting.
root@raspberrypi:/var/www/html/n#


Robert Vajda <robster0326@gmail.com> ezt írta (időpont: 2018. nov. 7., Sze,
11:03):

> I'm using 4 strain gauges connected to a Load Combinator, which is
> connected to a Load Cell Amplifier what is connected to my raspi. :( maybe
> the weight is too small?
>
> Robert Vajda <robster0326@gmail.com> ezt írta (időpont: 2018. nov. 7.,
> Sze, 10:41):
>
>> Yeps, i do everything as instructed by the program.
>>
>> NeonSpork <notifications@github.com> ezt írta (időpont: 2018. nov. 7.,
>> Sze, 9:31):
>>
>>> Just checking real quick, you're probably doing it right but it doesn't
>>> hurt to ask:
>>>
>>> You're making sure to have absolutely nothing at all on the scale until
>>> AFTER the program says "Place something with a known weight on the
>>> scale."
>>> right?
>>>
>>> On Wed, Nov 7, 2018, 09:20 robster <notifications@github.com> wrote:
>>>
>>> > Good morning!
>>> >
>>> > It still does not work for me, after downloading the new version:
>>> >
>>> >
>>> > root@raspberrypi:/var/www/html/n# python3 test.py
>>> > Initializing HX711
>>> > Place something with a known weight on the scale.
>>> > Press enter when ready.
>>> > 8366205
>>> > Enter weight on scale in grams: 434
>>> > Weight: -2.817254209046993g, verify that the measured weight is
>>> correct.
>>> > Provided the above result is correct the following can be used in your
>>> main
>>> > program:
>>> > Offset: 8420416.375
>>> > Scale: 19276.970046082948
>>> > Test complete. Exiting.
>>> >
>>> > I have checked everything, but can't find the root cause :(
>>> >
>>> > What else could i try?
>>> >
>>> > Jiri Dohnalek <notifications@github.com> ezt írta (időpont: 2018.
>>> nov. 4.,
>>> > V, 11:31):
>>> >
>>> > > Merged #5 <https://github.com/learn2develop/hx711py/pull/5> into
>>> master.
>>> > >
>>> > > —
>>> > > You are receiving this because you were mentioned.
>>> > > Reply to this email directly, view it on GitHub
>>> > > <https://github.com/learn2develop/hx711py/pull/5#event-1944758164>,
>>> or
>>> > mute
>>> > > the thread
>>> > > <
>>> >
>>> https://github.com/notifications/unsubscribe-auth/AEDj7r_U58N7wlz1tGUn8qzGPHstC3tAks5ursHvgaJpZM4YMtm_
>>> > >
>>> > > .
>>> > >
>>> >
>>> > —
>>> > You are receiving this because you authored the thread.
>>> > Reply to this email directly, view it on GitHub
>>> > <
>>> https://github.com/learn2develop/hx711py/pull/5#issuecomment-436541441>,
>>> > or mute the thread
>>> > <
>>> https://github.com/notifications/unsubscribe-auth/AfteJVC0tH6jUrGLsPzGrrKmcKzjAawgks5uspe6gaJpZM4YMtm_
>>> >
>>> > .
>>> >
>>>
>>> —
>>> You are receiving this because you were mentioned.
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/learn2develop/hx711py/pull/5#issuecomment-436544561>,
>>> or mute the thread
>>> <https://github.com/notifications/unsubscribe-auth/AEDj7rYqndaJpLRqDtVqhxX8U-ysuo4_ks5usppKgaJpZM4YMtm_>
>>> .
>>>
>>

NeonSpork commented:
I assumed as much but I had to ask! :D

Very strange. I can't think of anything particular at the moment, sorry!
You've tried with all three different gain settings?

Hopefully my new load sensor (4 of them like your set up) will arrive soon
and I can do some more rigorous testing.

On Wed, Nov 7, 2018, 11:05 robster <notifications@github.com> wrote:

> If i put more weight on it, the value change, but it's still negative:
>
> root@raspberrypi:/var/www/html/n# python3 test.py
> Initializing HX711
> Place something with a known weight on the scale.
> Press enter when ready.
> 8366255
> Enter weight on scale in grams: 4126
> Weight: -26.822929345328347g, verify that the measured weight is correct.
> Provided the above result is correct the following can be used in your main
> program:
> Offset: 8420312.625
> Scale: 2027.6914687348522
> Test complete. Exiting.
> root@raspberrypi:/var/www/html/n#
>
>
> Robert Vajda <robster0326@gmail.com> ezt írta (időpont: 2018. nov. 7.,
> Sze,
> 11:03):
>
> > I'm using 4 strain gauges connected to a Load Combinator, which is
> > connected to a Load Cell Amplifier what is connected to my raspi. :(
> maybe
> > the weight is too small?
> >
> > Robert Vajda <robster0326@gmail.com> ezt írta (időpont: 2018. nov. 7.,
> > Sze, 10:41):
> >
> >> Yeps, i do everything as instructed by the program.
> >>
> >> NeonSpork <notifications@github.com> ezt írta (időpont: 2018. nov. 7.,
> >> Sze, 9:31):
> >>
> >>> Just checking real quick, you're probably doing it right but it doesn't
> >>> hurt to ask:
> >>>
> >>> You're making sure to have absolutely nothing at all on the scale until
> >>> AFTER the program says "Place something with a known weight on the
> >>> scale."
> >>> right?
> >>>
> >>> On Wed, Nov 7, 2018, 09:20 robster <notifications@github.com> wrote:
> >>>
> >>> > Good morning!
> >>> >
> >>> > It still does not work for me, after downloading the new version:
> >>> >
> >>> >
> >>> > root@raspberrypi:/var/www/html/n# python3 test.py
> >>> > Initializing HX711
> >>> > Place something with a known weight on the scale.
> >>> > Press enter when ready.
> >>> > 8366205
> >>> > Enter weight on scale in grams: 434
> >>> > Weight: -2.817254209046993g, verify that the measured weight is
> >>> correct.
> >>> > Provided the above result is correct the following can be used in
> your
> >>> main
> >>> > program:
> >>> > Offset: 8420416.375
> >>> > Scale: 19276.970046082948
> >>> > Test complete. Exiting.
> >>> >
> >>> > I have checked everything, but can't find the root cause :(
> >>> >
> >>> > What else could i try?
> >>> >
> >>> > Jiri Dohnalek <notifications@github.com> ezt írta (időpont: 2018.
> >>> nov. 4.,
> >>> > V, 11:31):
> >>> >
> >>> > > Merged #5 <https://github.com/learn2develop/hx711py/pull/5> into
> >>> master.
> >>> > >
> >>> > > —
> >>> > > You are receiving this because you were mentioned.
> >>> > > Reply to this email directly, view it on GitHub
> >>> > > <https://github.com/learn2develop/hx711py/pull/5#event-1944758164
> >,
> >>> or
> >>> > mute
> >>> > > the thread
> >>> > > <
> >>> >
> >>>
> https://github.com/notifications/unsubscribe-auth/AEDj7r_U58N7wlz1tGUn8qzGPHstC3tAks5ursHvgaJpZM4YMtm_
> >>> > >
> >>> > > .
> >>> > >
> >>> >
> >>> > —
> >>> > You are receiving this because you authored the thread.
> >>> > Reply to this email directly, view it on GitHub
> >>> > <
> >>> https://github.com/learn2develop/hx711py/pull/5#issuecomment-436541441
> >,
> >>> > or mute the thread
> >>> > <
> >>>
> https://github.com/notifications/unsubscribe-auth/AfteJVC0tH6jUrGLsPzGrrKmcKzjAawgks5uspe6gaJpZM4YMtm_
> >>> >
> >>> > .
> >>> >
> >>>
> >>> —
> >>> You are receiving this because you were mentioned.
> >>> Reply to this email directly, view it on GitHub
> >>> <
> https://github.com/learn2develop/hx711py/pull/5#issuecomment-436544561>,
> >>> or mute the thread
> >>> <
> https://github.com/notifications/unsubscribe-auth/AEDj7rYqndaJpLRqDtVqhxX8U-ysuo4_ks5usppKgaJpZM4YMtm_
> >
> >>> .
> >>>
> >>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/learn2develop/hx711py/pull/5#issuecomment-436571444>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AfteJYvJrr-FDmVmp9i6UROoazAw2bzmks5usrBzgaJpZM4YMtm_>
> .
>

robster0326 commented:
Yes, tried with all 3 gain settings. It's the same. Slight difference, but
still negative value :(

Ok, thanks for your efforts. I hope you can do some testing.

NeonSpork <notifications@github.com> ezt írta (időpont: 2018. nov. 7., Sze,
11:08):

> I assumed as much but I had to ask! :D
>
> Very strange. I can't think of anything particular at the moment, sorry!
> You've tried with all three different gain settings?
>
> Hopefully my new load sensor (4 of them like your set up) will arrive soon
> and I can do some more rigorous testing.
>
> On Wed, Nov 7, 2018, 11:05 robster <notifications@github.com> wrote:
>
> > If i put more weight on it, the value change, but it's still negative:
> >
> > root@raspberrypi:/var/www/html/n# python3 test.py
> > Initializing HX711
> > Place something with a known weight on the scale.
> > Press enter when ready.
> > 8366255
> > Enter weight on scale in grams: 4126
> > Weight: -26.822929345328347g, verify that the measured weight is correct.
> > Provided the above result is correct the following can be used in your
> main
> > program:
> > Offset: 8420312.625
> > Scale: 2027.6914687348522
> > Test complete. Exiting.
> > root@raspberrypi:/var/www/html/n#
> >
> >
> > Robert Vajda <robster0326@gmail.com> ezt írta (időpont: 2018. nov. 7.,
> > Sze,
> > 11:03):
> >
> > > I'm using 4 strain gauges connected to a Load Combinator, which is
> > > connected to a Load Cell Amplifier what is connected to my raspi. :(
> > maybe
> > > the weight is too small?
> > >
> > > Robert Vajda <robster0326@gmail.com> ezt írta (időpont: 2018. nov. 7.,
> > > Sze, 10:41):
> > >
> > >> Yeps, i do everything as instructed by the program.
> > >>
> > >> NeonSpork <notifications@github.com> ezt írta (időpont: 2018. nov.
> 7.,
> > >> Sze, 9:31):
> > >>
> > >>> Just checking real quick, you're probably doing it right but it
> doesn't
> > >>> hurt to ask:
> > >>>
> > >>> You're making sure to have absolutely nothing at all on the scale
> until
> > >>> AFTER the program says "Place something with a known weight on the
> > >>> scale."
> > >>> right?
> > >>>
> > >>> On Wed, Nov 7, 2018, 09:20 robster <notifications@github.com> wrote:
> > >>>
> > >>> > Good morning!
> > >>> >
> > >>> > It still does not work for me, after downloading the new version:
> > >>> >
> > >>> >
> > >>> > root@raspberrypi:/var/www/html/n# python3 test.py
> > >>> > Initializing HX711
> > >>> > Place something with a known weight on the scale.
> > >>> > Press enter when ready.
> > >>> > 8366205
> > >>> > Enter weight on scale in grams: 434
> > >>> > Weight: -2.817254209046993g, verify that the measured weight is
> > >>> correct.
> > >>> > Provided the above result is correct the following can be used in
> > your
> > >>> main
> > >>> > program:
> > >>> > Offset: 8420416.375
> > >>> > Scale: 19276.970046082948
> > >>> > Test complete. Exiting.
> > >>> >
> > >>> > I have checked everything, but can't find the root cause :(
> > >>> >
> > >>> > What else could i try?
> > >>> >
> > >>> > Jiri Dohnalek <notifications@github.com> ezt írta (időpont: 2018.
> > >>> nov. 4.,
> > >>> > V, 11:31):
> > >>> >
> > >>> > > Merged #5 <https://github.com/learn2develop/hx711py/pull/5> into
> > >>> master.
> > >>> > >
> > >>> > > —
> > >>> > > You are receiving this because you were mentioned.
> > >>> > > Reply to this email directly, view it on GitHub
> > >>> > > <
> https://github.com/learn2develop/hx711py/pull/5#event-1944758164
> > >,
> > >>> or
> > >>> > mute
> > >>> > > the thread
> > >>> > > <
> > >>> >
> > >>>
> >
> https://github.com/notifications/unsubscribe-auth/AEDj7r_U58N7wlz1tGUn8qzGPHstC3tAks5ursHvgaJpZM4YMtm_
> > >>> > >
> > >>> > > .
> > >>> > >
> > >>> >
> > >>> > —
> > >>> > You are receiving this because you authored the thread.
> > >>> > Reply to this email directly, view it on GitHub
> > >>> > <
> > >>>
> https://github.com/learn2develop/hx711py/pull/5#issuecomment-436541441
> > >,
> > >>> > or mute the thread
> > >>> > <
> > >>>
> >
> https://github.com/notifications/unsubscribe-auth/AfteJVC0tH6jUrGLsPzGrrKmcKzjAawgks5uspe6gaJpZM4YMtm_
> > >>> >
> > >>> > .
> > >>> >
> > >>>
> > >>> —
> > >>> You are receiving this because you were mentioned.
> > >>> Reply to this email directly, view it on GitHub
> > >>> <
> > https://github.com/learn2develop/hx711py/pull/5#issuecomment-436544561>,
> > >>> or mute the thread
> > >>> <
> >
> https://github.com/notifications/unsubscribe-auth/AEDj7rYqndaJpLRqDtVqhxX8U-ysuo4_ks5usppKgaJpZM4YMtm_
> > >
> > >>> .
> > >>>
> > >>
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/learn2develop/hx711py/pull/5#issuecomment-436571444
> >,
> > or mute the thread
> > <
> https://github.com/notifications/unsubscribe-auth/AfteJYvJrr-FDmVmp9i6UROoazAw2bzmks5usrBzgaJpZM4YMtm_
> >
> > .
> >
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/learn2develop/hx711py/pull/5#issuecomment-436572263>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEDj7ifpWdUrOGGdlplJTeYBWZqhXTaSks5usrEdgaJpZM4YMtm_>
> .
>
